src/stavenote.ts

```diff
@@ -1207,6 +1211,12 @@ export class StaveNote extends StemmableNote {
       this.setStem(new Stem(stemOptions));
     }
 
+    // If we will render a flag, we shorten the stem so that the tip
+    // does not poke through the flag.
+    if (this.shouldDrawFlag() && this.stem) {
+      this.stem.renderHeightAdjustment = -3;
```

Comment:
Can we move this constant into a font metrics file?

Reply:
I was just sticking to the convention of keeping magic numbers (like +/- 2) in the rendering functions. 😂

But yes, I'll find a way to move these adjustments out to the metrics files. This will future proof the code in case we add more fonts.

Reply:
Done.